iot-hub/Samples/device/ConventionBasedOperations/Thermostat/ThermostatSample.cs

```diff
@@ -277,5 +286,24 @@ private static double GenerateTemperatureWithinRange(int max = 50, int min = 0)
         {
             return Math.Round(s_random.NextDouble() * (max - min) + min, 1);
         }
+
+        private async Task UpdateForEmptyWritableProperty(CancellationToken cancellationToken)

```

Comment:
consider passing a param with the value to initialize

Reply:
added